apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -42,9 +42,13 @@ export function SenderRecipient({
             </Heading>
             <div className="flex flex-col gap-[15px] justify-center relative">
                 {singleTransferCoin && (
-                    <div className="absolute border-2 border-sui-steel overflow-y-hidden h-[calc(55%)] w-4 border-r-transparent border-t-transparent mt-1 ml-1.5 rounded-l border-dotted" />
+                    <div className="absolute border-dashed border border-sui-steel overflow-y-hidden h-[calc(57%)] w-4 border-r-transparent border-t-transparent mt-1 ml-1.5 rounded-l" />
                 )}
-                <SenderRecipientAddress isSender address={sender} />
+                <SenderRecipientAddress
+                    isSender
+                    address={sender}
+                    data-testid="transaction-sender"
```

Comment:
Unless you spread props, this won't appear on the rendered element. This probably belongs either in a wrapper, or in the component definition.